test/contract/pact/probate/external/FeesRegisterClient.spec.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * The following example is for Pact version 5
+ */
+const path = require('path');
+const chai = require('chai');
+const {Pact, Matchers} = require('@pact-foundation/pact');
+const {somethingLike, like} = Matchers;
+const chaiAsPromised = require('chai-as-promised');
+const FeeLookupClient = require('app/services/FeesLookup');
+const config = require('config');
+const getPort = require('get-port');
+const assert = chai.assert;
+chai.use(chaiAsPromised);
+let headers;
+
+describe('Pact FeesRegisterClient', () => {
+
+    let MOCK_SERVER_PORT;
+    let provider;
+    getPort().then(portNumber => {
+        MOCK_SERVER_PORT = portNumber;
+        // (1) Create the Pact object to represent your provider
+        provider = new Pact({
+            consumer: 'probate_frontend',
+            provider: 'feeRegister_lookUp',
+            port: MOCK_SERVER_PORT,
+            log: path.resolve(process.cwd(), 'logs', 'pactFeesRegister.log'),
+            dir: path.resolve(process.cwd(), config.services.pact.pactDirectory),
+            logLevel: 'INFO',
+            spec: 2
+        });
+    });
+
+    const ctx = {
+        sessionID: 'someSessionId',
+        authToken: 'authToken',
+        session: {
+            serviceAuthorization: 'someServiceAuthorization'
+        }
+    };
+
+    const issuesData = {
+        amount_or_volume: 2500000,
+        applicant_type: 'personal',
+        channel: 'default',
+        event: 'issue',
+        jurisdiction1: 'family',
+        jurisdiction2: 'probate registry',
+        service: 'probate'
+    };
```

Comment:
Note: issueData is now based on feature toggle ft_newfee_register_code: 'probate-newfee-register-code'. when the feature toggle is ON, it will expect additional keyword:PA for amount more than 5000 and PAL5K for amount less than 5000.  for details of data expected please look at default.yml for feesRegister.newfee_issuesData. and feeRegister.newfee_issuesDataIhtMinAmount